tests/assert/assert.go

```diff
@@ -21,3 +21,13 @@ func Error(t *testing.T, err error, expected string) {
 		t.Fatalf("expected '%s' but got '%s'", expected, err.Error())
 	}
 }
+
+// True asserts that an interface is equal to boolean "true"
+func True(t *testing.T, expected interface{}) {
```

Comment:
Likewise here, let's fix-up this signature and then we can drop-in testify under a separate PR.